apis/controller/v1alpha1/sliceconfig_types.go

```diff
@@ -80,10 +80,10 @@ type WorkerSliceGatewayProvider struct {
 
 type SliceGatewayServiceType struct {
 	// +kubebuilder:validation:Required
-	Cluster string `json:"cluster,omitempty"`
```

Comment:
I think we should not remove "omitempty". If this config is not specified, we assume that it is NodePort.

Reply:
Currently the behaviour is, if user is defining an array element then both `cluster` & `type` is required else we get `NodePort` for all participating clusters. Please correct me if I understood the comment incorrectly.